tests/test_instalabel.py

```diff
@@ -0,0 +1,50 @@
+import shippo
+from shippo.models.components import Carriers, DistanceUnitTemplate, WeightUnit, \
+    AddressCreateRequest, ShipmentCreateRequest, InstantTransactionRequestBody, \
+    ParcelCreateRequest
+from tests.helpers_custom import get_carrier_account
+
+
+class TestInstalabel:
+
+    def test_instalabel(self, api: shippo.Shippo):
+        carrier_account = get_carrier_account(api, Carriers.USPS)
+
+        create_transaction_response = api.transactions.create_transaction(
+            request_body=InstantTransactionRequestBody(
+                carrier_account=carrier_account.object_id,
+                servicelevel_token="usps_ground_advantage",
+                shipment=ShipmentCreateRequest(
+                    address_from=AddressCreateRequest(
+                        name="Rachael",
```

Comment:
are we using the static values to match the documented use case?

Reply:
yeah